Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2770,22 +2768,21 @@ - (void)imagePickerController:(UIImagePickerController *)picker
     }
 }
 
-- (void)sendMessageAttachment:(SignalAttachment *)attachment
+- (void)sendMessageAttachments:(NSArray<SignalAttachment *> *)attachments
 {
     OWSAssertIsOnMainThread();
-    // TODO: Should we assume non-nil or should we check for non-nil?
-    OWSAssertDebug(attachment != nil);
-    OWSAssertDebug(![attachment hasError]);
-    OWSAssertDebug([attachment mimeType].length > 0);
-
-    OWSLogVerbose(@"Sending attachment. Size in bytes: %lu, contentType: %@",
-        (unsigned long)[attachment dataLength],
-        [attachment mimeType]);
+    for (SignalAttachment *attachment in attachments) {
+        // TODO: Should we assume non-nil or should we check for non-nil?
```

Comment:
`NSArray` can't hold nil; we can assume non-nil.

Reply:
Good point.